tests/Etsy/EtsyApiBuildRequestTest.php

```diff
@@ -76,7 +76,7 @@ public function testValidData()
 				"tags" => array('fashion, othertag'),
 				"who_made" => "collective",
 				"is_supply" => true,
-				"when_made" => "2010_2015",
+				"when_made" => "2010_2016",
```

Comment:
Sounds Good

On Thu, Sep 1, 2016 at 12:39 PM Iñaki Abete <notifications@github.com>
wrote:

> In tests/Etsy/EtsyApiBuildRequestTest.php
> <https://github.com/inakiabt/etsy-php/pull/13#discussion_r77216571>:
>
> > @@ -76,7 +76,7 @@ public function testValidData()
> >  				"tags" => array('fashion, othertag'),
> >  				"who_made" => "collective",
> >  				"is_supply" => true,
> > -				"when_made" => "2010_2015",
> > +				"when_made" => "2010_2016",
>
> In order to not have to change this every year, let's change this value to
> 2000_2009. What do you think?
>
> —
> You are receiving this because you authored the thread.
> Reply to this email directly, view it on GitHub
> <https://github.com/inakiabt/etsy-php/pull/13/files/b7189dc554310d15f481bb072ef40019dbca9383#r77216571>,
> or mute the thread
> <https://github.com/notifications/unsubscribe-auth/AFSc5UFi0IKpOA3kl5Kb2PR5ZfOAmDvkks5qlwnagaJpZM4Jy7Il>
> .
>